src/Core/Processor/ImageProcessor.php

```diff
@@ -156,11 +171,33 @@ protected function generateSize(OutputImage $outputImage): array
             $size .= $preserveNaturalSize ? '\>' : '';
             $gravity = '';
         }
-        //In cas on png format, remove extent option
-        if ($outputImage->isOutputPng()) {
-            $extent = '';
-        }
 
         return [$size, $extent, $gravity];
     }
+
+    /**
+     * @return array
+     */
+    protected function getSourceImageDimensions()
```

Comment:
where those functions `getSourceImageDimensions ` and `getSourceImageInfo ` called ?

Reply:
These are called in ImageProcessor